client/src/key_management/bundle.rs

```diff
@@ -84,22 +84,22 @@ impl KeyBundleContent {
     }
 
     pub fn deserialize(bundle: PrimitiveKeyBundleContent) -> KeyBundleContent {
-        let one_time_key: Option<(PreKeyId, PublicKey)> =
-            if (bundle.onetime_public_key_id.is_none()
-                || bundle.onetime_public_key.is_none()
-                || bundle
-                    .onetime_public_key_id
-                    .expect("One time pre key id is somehow None after a none check - kill me")
-                    == 0)
-            {
-                None
-            } else {
-                Some((
-                    bundle.onetime_public_key_id.expect("").into(),
-                    PublicKey::deserialize(&(*bundle.onetime_public_key.to_owned().expect("")))
-                        .expect(""),
-                ))
-            };
+        let one_time_key: Option<(PreKeyId, PublicKey)> = if (bundle
+            .onetime_public_key_id
+            .is_none()
+            || bundle.onetime_public_key.is_none()
```

Comment:
This should work, right?
```suggestion
            || bundle
            .onetime_public_key
            .is_none()
```